estate/__manifest__.py

```diff
@@ -0,0 +1,13 @@
+{
+    'name': "Real estate",
+    'depends': ['base'],
+    'application': True,
+    'installable': True,
+    'data': [
+        'security/ir.model.access.csv',
+        'views/estate_menus.xml',
+        'views/estate_property_views.xml'
```

Comment:
We usually add a comma at the end of every line for multiline dicts, lists, arguments, tuples etc. 
This is to avoid the next person that wants to add a new line after yours would have to add a diff to your line (which will set him as author of the line when someone will run git blame)
```suggestion
        'views/estate_property_views.xml',
```